HedvigInsurance/web-onboarding
# fix(GRW-642): disable start date when current insurance automatic cancellation is requested

## What?

Disable `StartDate` field in case users request the cancellation of their old insurance

## Why?

In such situations the start date should be right after the end date of the old insurance, which means there's no need to let them pick a start date.

**[Ticket](https://hedvig.atlassian.net/browse/GRW-642)**

## Screenshots / recordings 

<img width="1219" alt="Screenshot 2022-02-10 at 09 18 59" src="https://user-images.githubusercontent.com/19200662/153366280-7fb614ba-8f2c-496d-bc01-e6eba7248c2a.png">
<img width="1134" alt="Screenshot 2022-02-28 at 11 47 32" src="https://user-images.githubusercontent.com/19200662/155970155-4cbe3bb6-b8bc-4939-b872-cc3a672b42b1.png">

## How to test it

Access the review app linked below and complete the switcher flow in _Embark_:

1. Select an insurance provider that we're able to cancel insurances from, E.g: Trygghasna. at the beginning of the flow;
2. Click on [Skip] button and finish the flow;
<img width="421" alt="Screenshot 2022-02-03 at 14 34 26" src="https://user-images.githubusercontent.com/19200662/152353008-8166e1dd-a62e-4e33-9af8-966f8ed0bdf2.png">
3. You should not be able to interact with Date picker fields for current insurances that should be cancelled by Hedvig while signing a new insurance. Please note that you can control which insurance should be cancelled by interacting with its corresponding _Cancel Insurance_ checkbox.

### [Review app](https://web-onboardi-fix-grw-64-r7mmz7.herokuapp.com/se-en/new-member)
